Testcase/lvv-t2213 - Look-up Table Application from MTMount Elevation and Azimuth Changes

b1quint opened the pull request:
Updated the notebooks to run and analyze the data for LVV-T2213. The execution notebook seems to be in a final state. The data-analysis notebook might need some improvements in the future but, for now, the current content should be sufficient.

I ran this notebook on TTS and I do have some questions regarding the compensated position of the hexapods. It is not clear to me yet when they should be published.

b1quint committed:
Rename data-analysis notebook for LVV-T2213

b1quint committed:
Add notebook to run LVV-T2242 and updated vandv python
- Create notebook that runs LVV-T2241
- Create notebook to analyse data for the previous notebook
- Create the vandv.mtaos module to plot Degrees of Freedom
- Update vandv.info to generate an index for the Script CSC using the
  test case id and the date.

b1quint committed:
Clean up notebook that runs LVV-T2213

b1quint committed:
Update notebooks with plots for LVV-T2213

isotuela commented on notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:
```diff
@@ -391,37 +257,24 @@
    "source": [
-    "# We are running this command because we do not want to track\n",
-    "await mtcs.rem.mtmount.cmd_moveToTarget.set_start(azimuth=0, elevation=90)"
+    "await mtcs.point_azel(az=0, el=90)"
```
Move in small steps to the zenith, similar to the  `safe` = False` option below.

b1quint replied:
Done.

isotuela commented on notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:
```diff
@@ -1125,26 +653,27 @@
+    "    \n",
+    "    print(datetime.now())\n",
+    "    script.log.info(f\"MOVING - {test_case} {test_exec} Moving to az/el = 180./85.\")\n",
+    "    await mtcs.point_azel(az=180, el=85)\n",
```
Why changing the elevation to 85 degrees, as well?

b1quint replied:
I don't know. This comes from the test case.

isotuela commented on notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:
```diff
@@ -1201,10 +710,11 @@
     "# Specially if running at TTS\n",
-    "target = mtcs.radec_from_azel(az=0, el=80)\n",
+    "if safe:\n",
+    "    target = mtcs.radec_from_azel(az=0, el=80)\n",
```
Please add the `not Safe` option

b1quint replied:
Done.

isotuela commented on notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:
```diff
@@ -0,0 +1,897 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.slew_object(target, rot_type=RotType.PhysicalSky, rot=1.9)"
```
Perhaps add here safe = False to move mount in small steps?

b1quint replied:
This is a stray commit that comes from another test case. This change should not be here.

isotuela commented on notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:
```diff
@@ -0,0 +1,897 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.point_azel(az=0, el=80)\n",
```
Same here. Perhaps add here safe = False to move mount in small steps?

b1quint replied:
Same here. This is a stray commit. It should be updated on another branch/PR.

isotuela reviewed (commented):
It looks good. I just noted some instances in the execution notebooks for LVV-T2213 and LVV-T2241 in which the `safe=False` option to move the mount in small steps could be added for added safety.

b1quint committed:
Update notebook to run LVV-T2213 with comments in the PR

- Add non-safe case in the Close Up session
- Set M1M3 to STANDBY instead of OFFLINE in Close Up session
- Consider safe/unsafe options in the Setting Up session

isotuela reviewed (approved):
Thank you so much. I like the added Safer option.
